pkg/server/router/cloud.go

```diff
@@ -78,3 +80,27 @@ func (router *router) pingCloud(request *restful.Request, response *restful.Resp
 
 	response.WriteHeaderAndEntity(http.StatusOK, resp)
 }
+
+// listClouds handles the request to list all clouds.
```

Comment:
// listWorkers handles the request to list all workers.

Reply:
done